### ISSUE TITLE
Reason why?

### ISSUE THREAD

**Issue by cvladan** [webinstall/webi-roadmap#9]
I don't quite understand the purpose of yet another alternative when we already have tools like Scoop, Winget, and Choco. Don't these tools already cover all use cases?

To me this seems to be almost exactly the same as [Scoop](https://scoop.sh/).
Can someone explain diffs to me?

**Comment by cvladan** [comment]
1. Multi-platform
2. It requires no additional installations, as you copy-paste a single command from the web. So it's great for one-time setup apps.

Next?

**Comment by coolaj86** [comment]
Ryan and I created it because we work with a lot of clients and a not of new developers and we need a streamlined way to get things set up very quickly with minimal headache.

I wanted to add Windows support because a lot of people aren't going to go full-on Linux or Mac just to dip their toe in development, but I still want them to have the best possible tools available to get started.

From the homepage:

# Why Webi?

> Install (mostly) cross-platform tools, from official builds

- quickly
- without sudo or Admin
- without a package manager
- without changing system file permissions

The FAQ on "Why Webi?" has a little more detail too:
https://webinstall.dev/faq/#why

In summary:

> So keep your package manager, and use it for the things it does well. Use webi when it helps - or when the alternative hurts.

**Comment by coolaj86** [comment]
Closing since there's no action item.
